client/auth/signup.tsx

```diff
@@ -316,22 +316,24 @@ export function Signup() {
               {...bindCheckable('policyAgreement')}
               label={
                 <span>
-                  {t('auth.signup.readAndAgree', 'I have read and agree to the')}{' '}
-                  <DialogLink
-                    dialogType={DialogType.TermsOfService}
-                    text={t('auth.signup.termsOfServiceLink', 'Terms of Service')}
-                  />
-                  ,{' '}
-                  <DialogLink
-                    dialogType={DialogType.AcceptableUse}
-                    text={t('auth.signup.acceptableUseLink', 'Acceptable Use')}
-                  />
-                  , and{' '}
-                  <DialogLink
-                    dialogType={DialogType.PrivacyPolicy}
-                    text={t('auth.signup.privacyLink', 'Privacy')}
-                  />{' '}
-                  policies
+                  <Trans t={t} i18nKey='auth.signup.readAndAgree'>
+                    I have read and agree to the{' '}
+                    <DialogLink
```

Comment:
This is really all the same string, so you probably need to change `DialogLink` to take this string as a child instead